pnpm-lock.yaml

```diff
@@ -73,13 +73,13 @@ importers:
         version: link:monorepo/mytsup
       tsc-absolute:
         specifier: ^1.0.1
-        version: 1.0.1(typescript@5.4.2)
+        version: 1.0.1(typescript@5.4.4)
```

Comment:
Noticed lot of changes here, seems like its to be expected though?

Reply:
Yeah, I deduped.